src/common/graphcmn.cpp

```diff
@@ -911,13 +915,38 @@ wxGraphicsBitmap wxGraphicsContext::CreateSubBitmap( const wxGraphicsBitmap &bmp
 }
 #endif
 
-#ifdef __WXMSW__
-#if wxUSE_ENH_METAFILE
+#if defined(__WXMSW__) && wxUSE_ENH_METAFILE
 /* static */ wxGraphicsContext* wxGraphicsContext::Create( const wxEnhMetaFileDC& dc)
 {
     return wxGraphicsRenderer::GetDefaultRenderer()->CreateContext(dc);
 }
 #endif
+
+#ifndef wxNO_RTTI
+wxGraphicsContext* wxGraphicsContext::CreateFromUnknownDC(const wxDC& dc)
+{
+    const wxWindowDC *windc = dynamic_cast<const wxWindowDC*>(&dc);
```

Comment:
Pretty minor but I'd put the variable declaration inside the `if` (here and in all the other cases below, of course).